src/qt/bitcoin.cpp

```diff
@@ -381,7 +398,7 @@ void BitcoinApplication::startThread()
     if(coreThread)
         return;
     coreThread = new QThread(this);
-    BitcoinCore *executor = new BitcoinCore();
+    executor = new BitcoinCore();
     executor->moveToThread(coreThread);
```

Comment:
It's not allowed to use this object after moving it to another thread.
I'd suggest calling `InitParameterInteraction()` directly from `BitcoinApplication::parameterSetup()` and not bothering with the stuff in BitcoinCore.